CHANGELOG

```diff
@@ -1,3 +1,7 @@
+Version 0.3.1 (2019-09-26)
```

Comment:
Updates to the `CHANGELOG` should be done in a separate `Prepare for release` commit, which will be the only commit that does not close a corresponding issue in the issue tracker and is not noted in the `CHANGELOG`.